abracadabra/src/g4-mandatory/trajectory_point.cc

```diff
@@ -0,0 +1,24 @@
+#include "trajectory_point.hh"
+
+G4Allocator<trajectory_point> TrjPointAllocator;
+
+trajectory_point::trajectory_point()
+: position_(0.,0.,0.)
+, time_(0.) {}
```

Comment:
Let's avoid the HEP-mythology of putting trailing dots on `double` literals (though braced-initialization *sometimes* forces us to).

`position_` is a `G4ThreeVector` so its default constructor already fills it with zeros ... but if you think emphasis on the zeros helps in reading the code, then keep them.